src/capybara.js

```diff
@@ -269,8 +269,20 @@ Capybara = {
     return true;
   },
 
+  sendKeys: function (index, keys) {
+    var strindex, length;
+
+    length = keys.length;
+
+    this.focus(index);
```

Comment:
Not quite sure if I follow. If you don't focus the element, the keys will be sent to the one that is focused. But i agree it should first check if their is any keys to press (send) at all. This should mimic the user behaviour more closely. 

How do you mean use this shared implementation? If it is deep inside set it would be hard, no?
Also, the semantics are different, you don't always want to clear the value, and you can send key presses to _any_ element, not just form input.